index.html

```diff
@@ -44,7 +44,8 @@ <h2>GAME OVER!</h2>
       </section>
     </main>
     <footer class="game-footer">
-      <p>Made for Hacktoberfest by these <strong><a href="https://github.com/itaditya/got-death-quiz#contributors" target="_blank">awesome people.</a></strong></p>
+      <p>Made for Hacktoberfest by these <strong><a href="https://github.com/itaditya/got-death-quiz#contributors" target="_blank">awesome people.</a></strong><a href="https://twitter.com/intent/tweet?text=Game%20of%20Thrones%20Death%20Quiz&url=https%3A%2F%2Fgot-death-quiz.netlify.com&hashtags=roadtohacktober,hacktoberfest" target="blank" rel="noopener noreferrer"><img
```

Comment:
Put the Twitter button below the "Reload to play again" section. Also in the text of the tweet we should have the user's score which you can get from the `store.score`.

So basically by default your twitter will be hidden just like other game-over stuff and doesn't have to contain any link. Once game over, generate the link with game score included and rest of the text you have and insert that link in the Tweet anchor tag and then un-hide it.

Reply:
Completed in new commit